product-scenarios/1-manage-public-partner-private-apis/1.1-expose-service-as-rest-api-and-apply-qos/common-utils/src/main/java/org/wso2/am/scenario/test/common/APIRequest.java

```diff
@@ -49,6 +72,14 @@ public void init() {
         this.addParameter("visibility", this.visibility);
         this.addParameter("version", this.version);
         this.addParameter("swagger", this.swagger);
+        if(tiersCollection != null)
+        {
```

Comment:
Shall we fix formatting issues

Reply:
Fix it in dc9a5c9555745294c47d4dc0e6cad63bdf5e4410